### PR TITLE
Removed text from the "Free for per..." section</p>

### PR DESCRIPTION
**Pull request by SirSamTumless**
Removed <p>See the table below to find the plan that’s right for you</p> as per the copy doc https://docs.google.com/document/d/1So3n5HEpBX39xxEkkChnve09gKSr9TgYUcbfdOTCevk/edit

## Done

- [List of work items including drive-bys.]

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes #

## Screenshots

[If relevant, please include a screenshot.]

### CODE REVIEW

**Review by ClementChaumel — APPROVED** [review]

**Comment by ClementChaumel** [comment]
image
Looks like you missed the text next to the button

File: templates/advantage/index-no-login.html [hunk 1]
```diff
@@ -54,13 +53,13 @@ <h3>Per-machine enterprise subscriptions</h3>
             });">
               Buy
           </a>
+           <p>See the table below to find the plan that’s right for you</p>
           <span>an enterprise subscription</span>
         </p>
```
Comment (ClementChaumel) [inline templates/advantage/index-no-login.html:58]:
```suggestion
          <span>an enterprise subscription</span>
        </p>
        <p class="u-no-margin--bottom">See the table below to find the plan that’s right for you</p>
```

File: templates/advantage/index-no-login.html [hunk 2]
```diff
@@ -40,7 +40,6 @@ <h5>
       <div class="col-6">
         <h3>Per-machine enterprise subscriptions</h3>
         <p>Enterprises choose Ubuntu Advantage for extended security, certified compliance and 24x7support.</p>
-        <p>See the table below to find the plan that’s right for you</p>
         <p class="u-no-margin--bottom">
```
Comment (ClementChaumel) [inline templates/advantage/index-no-login.html:43]:
```suggestion
        <p>
```

**Review by ClementChaumel — COMMENTED** [review]
I've left you some suggestions that _should_ produce the required result!

**Review by ClementChaumel — APPROVED** [review]
I love approving my own code. Outstanding work :+1:

**Comment by ClementChaumel** [comment]
@anasereijo since you're back do you mind having a quick look? image

**Comment by anasereijo** [comment]
Outstanding work! can you just add a full stop at the end of the sentence?

**Comment by SirSamTumless** [comment]
Why is this not showing on the live site?

**Pull request merged**